lib/nats.js

```diff
@@ -1219,7 +1217,7 @@ Client.prototype.publish = function(subject, msg, opt_reply, opt_callback) {
         }
         this.sendCommand(psub + Buffer.byteLength(str) + CR_LF + str + CR_LF);
     } else {
-        var b = new Buffer(psub.length + msg.length + (2 * CR_LF_LEN) + msg.length.toString().length);
+        var b = Buffer.alloc(psub.length + msg.length + (2 * CR_LF_LEN) + msg.length.toString().length);
```

Comment:
Does it have a significant performance impact? The counterpart of `new Buffer` is `Buffer.allocUnsafe`